ansible/roles/linux-user/files/user-create-playbook.yml

```diff
@@ -77,7 +77,7 @@
 
     - name: Ensure Azimuth home directory has the correct permissions
```

Comment:
I think we should completely remove this task, or at least remove `recursive: true`.

IMO its reasonable to expect a user to set permissions on files in their home directory, and we shouldn't come along and just set them all to `0750` indiscriminately - it could expose secrets (`0600`) to members of the same group.

Reply:
The home directory is created with the correct permissions on line 60, I think deleting this makes sense.

I don't think what you describe will happen, because this only runs once, shortly after creating the azimuth user. But equally, I think is must now be a noop?

@assumptionsandg do you remember why we needed this in the first place? or was it historic?

Reply:
I tried removing the task completely at first and then wasn't able to SSH into the VM at all. My guess was that dropping the task meant the azimuth user's authorised keys file then ended up with permissions that SSH wasn't happy with or something like that.

Reply:
This will run on a reimage/upgrade right? After a user has created all their files etc

Removing `recurse: true` might be enough...

Reply:
Im building a separate image over [here](https://github.com/azimuth-cloud/azimuth-images/pull/353) to try to figure out why removing that task completely didn't work

Reply:
For some reason removing that task now seems to just work... I think I must have been testing two fixes at once previously and it was the other one that was messing up the SSH access, or there might have just been some other cloud level issue in the mix 🤷‍♂️